stock_by_warehouse/views/product_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="product_normal_form_view" model="ir.ui.view">
+        <field name="name">product.product.form.stock.warehouse</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='default_code']" position="before">
+                <field name="warehouses_stock"
+                    widget="warehouse"
+                    groups="stock.group_stock_multi_locations"/>
+                <field name="warehouses_stock_location"
+                    widget="warehouse"
+                    options='{"by_location": True}'
+                    colspan="1"
+                    groups="stock.group_stock_multi_locations"/>
```

Comment:
@agarzaarvizu that's because now the group [is automatically set when creating warehouses](https://github.com/odoo/odoo/commit/8414e6b9fde7), but that doesn't mean the group doesn't exist anymore.

I would advice to ask before applying this kind of changes.

Reply:
In that case, how can I solve that problem? Cause there is no group_stock_multi_warehouse and the view depends on it.

Reply:
Hi @agarzaarvizu The group is still [there](https://github.com/odoo/odoo/blob/15.0/addons/stock/security/stock_security.xml#L15), it is giving you a traceback?

Reply:
@CarmenMiranda No, just the view doesn't appears even if I activate the multi locations option in the settings.